.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,208 @@
+# Contributing
+
+<img align="right" width="25%" style="margin-bottom: 2em" src="https://owlbert.io/images/owlberts-png/camp.png.png">
+
+## Commit Conventions
+
+For our general commit conventions please consult our organization contributing guidelines [here](https://github.com/readmeio/.github/blob/main/.github/CONTRIBUTING.md).
+
+## Help
+
+If you need help don't be afraid to open up an issue!
+
+## Packages
+### `httpsnippet-client-api` snippet
+
+If you wish to add or change functionality to how `api` snippets are generated, your changes should go within the `httpsnippet-client-api` package in `packages/`. We have a nifty test suite set up there that'll run pre-made JS code snippets for the [api](https://npm.im/api) package within a virtual environment, with mocked out HTTP requests, to ensure that everything is well and good. It's pretty neat!
+
+To add a new test into this suite you should do the following:
+
+1. Create a new directory for your test in `packages/httpsnippet-client-api/test/__datasets__`. This directory should be a short and clear description of what you're testing.
+    * If you need to give additional context to your test please do add a `README` in this directory!
+2. In this new directory you'll need to create 4 files: `har.json`, `openapi.json`, `mock.json`, and `output.js`.
+
+#### har.json
+
+The way that `httpsnippet-client-api` works is that it takes a HAR object and constructs a snippet according to it, and what the matching operation in an OpenAPI definition, say the API request should function.
```

Comment:
This reads a bit weirdly so I tried rewriting it, but feel free to mess with it further:
```suggestion
`httpsnippet-client-api` works by taking a HAR object and its matching operation in an OpenAPI definition and constructing a code snippet based upon them.
```